web/models/meeting_type.ex

```diff
@@ -7,14 +7,16 @@ defmodule Meetings.MeetingType do
     field :title, :string
     field :description, :string
     field :location, :string
+    field :building, :string
```

Comment:
Alright, I'll move building and website information over to extras so that the MeetingType model doesn't need to change. We can get rid of this branch. I'll rebranch with prettified json for the next pull request.